docs/source/en/api/pipelines/kolors.md

```diff
@@ -41,6 +41,64 @@ image = pipe(
 image.save("kolors_sample.png")
 ```
 
+## IP Adapter
+
+Kolors needs a different IP Adapter to be able to work, also it uses Openai-CLIP-336 as an image encoder.
```

Comment:
Maybe link to the model here https://huggingface.co/openai/clip-vit-large-patch14-336

```suggestion
Kolors needs a different IP Adapter to work, and it uses Openai-CLIP-336 as an image encoder.
```